examples/extensions/uniffi/JsonObject.toml

```diff
@@ -0,0 +1,19 @@
+# Example of wrapping on the both sides.
+
+wrapped_type = "String"
+
+[rust]
+imports = ["serde_json"]
+convert_from_wrapped = "serde_json::from_string({})"
+convert_to_wrapped = "{}.to_string()"
+
+
+[kotlin]
+imports = ["org.json.JSONObject"]
+convert_from_wrapped = "JSONObject({})"
+convert_to_wrapped = "{}.toString()"
+
+[python]
+imports = ["json"] 
+convert_from_wrapped = "json.loads({})"
+convert_to_wrapped = "json.dumps({})"
```

Comment:
I originally named these keys `wrap` and `unwrap`, but it felt a bit confusing in this case.  Is the string wrapping JSON or is JSON wrapping the string?  These names felt more clear to me.

Reply:
For me, it pointed my eyes to the `wrapped_type` key which cleared things up.  I don't think the naming is perfect though -- definitely open to suggestions.  On that front, maybe `adapter` is more clear than `wrapper`?  I'm not really sure.

Reply:
> For me, it pointed my eyes to the `wrapped_type`

I think that's exactly why I don't mind `wrap` and `unwrap`. I agree a nicer name would be better and I guess `to_wrapped`/`from_wrapped` is a bit more explicit, but a tiny bit uglier IMO :) I don't really care.

> I think I could get behind the words `from` and `into`

`from_primitive` and `to_primitive` might work?